_posts/2024-10-11-tiered-cache.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Tiered caching in OpenSearch"
+authors:
+   - upasagar
+   - akjain
+   - kkhatua
+   - kolchfa
+date: 2024-10-11
+categories:
+  - technical-posts
+has_science_table: true
+meta_keywords: OpenSearch, tiered caching, query performance, on-heap cache, disk-based cache, request cache, cache hit ratio, OpenSearch Benchmark, performance optimization, caching
+meta_description: Learn how tiered caching in OpenSearch enhances query performance by combining on-heap and disk-based caching. Explore its benefits, limitations, and practical use cases in this detailed blog.
```

Comment:
Updated meta: 
_meta_keywords:_ tiered caching, disk-based caching, on-heap caching, OpenSearch caching performance, how tiered caching works 

_meta_description:_ Explore how OpenSearch combines on-heap and disk-based caching to handle larger datasets and improve performance. Learn about the trade-offs of tiered caching, how it works, and future developments.